tests/fsharp/Compiler/Language/OptionalInteropTests.fs

```diff
@@ -163,3 +163,86 @@ Test.OverloadedMethodTakingNullableOptionals(x = 6) |> ignore
         let fs = Compilation.Create(fsSrc, CompileOutput.Exe, options = [| $"--langversion:{langVersion}" |], cmplRefs = [cs])
         CompilerAssert.Compile fs
 
+    // Tests for issue #19074: Support voption with optional parameter syntax (?x=)
```

Comment:
...Why is it even putting tests in this file at all? This file is about interop with parameters marked with `System.Runtime.InteropServices.OptionalAttribute`, but https://github.com/dotnet/fsharp/issues/19074 is about F#-style optional parameters.